blueprints/banking_blueprint/__init__.py

```diff
@@ -0,0 +1,475 @@
+# -*- coding: utf-8 -*-
+"""This module contains a MindMeld application"""
+from mindmeld import Application
+from mindmeld.core import FormEntity
+import random
+import json
+import re
+
+app = Application(__name__)
+
+__all__ = ["app"]
+
+# User data and helper functions for handlers
+
+user_data = {}
+
+sample_users = {
+    "johndoe123": 0,
+    "larry_l12": 1,
+    "splashbro30": 2,
+}
+
+
+def _pull_data(request):
+    with open("banking_blueprint/data/sample_user_data.json") as f:
```

Comment:
We should use relative path like this:
```suggestion
    with open("./banking_blueprint/data/sample_user_data.json") as f:
```

Or better, compute the path directly from file:
```
import os
directory = os.path.dirname(os.path.abspath(__file__))
path = os.path.join(directory, "banking_blueprint/data/sample_user_data.json")